cms/src/main/java/uk/nhs/digital/arc/process/ManifestProcessingSummary.java

```diff
@@ -0,0 +1,30 @@
+package uk.nhs.digital.arc.process;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ManifestProcessingSummary {
+    private List<ProcessOutcome> outcomes = new ArrayList<ProcessOutcome>();
```

Comment:
You can reduce the verbosity of this List to private `List<ProcessOutcome> outcomes = new ArrayList<ProcessOutcome>(); `as the compiler will infer the type